runtime/shared_common/shrinit.cpp

```diff
@@ -3952,6 +3953,30 @@ j9shr_getCacheSizeBytes(J9JavaVM *vm)
 	return ret;
 }
 
+/**
+ * Determine the type of shared class cache that is enabled. Either the current default, 
```

Comment:
"Either the current default, Bootstrap Classes Only, or user defined shared cache" should be something like:
"Either the current default (Bootstrap Classes Only), or user defined shared cache"